config/docker/cros-tast.jinja2

```diff
@@ -20,8 +20,8 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 
 {% include 'fragment/cros-lava.jinja2' %}
 # Update gsutil to latest version, so it doesnt download it in LAVA
-# docker each time
-RUN gsutil update -n
+# docker each time. But don't fail if we are already latest version
+RUN gsutil update -n || true
```

Comment:
This will in general silent any error which can occur during gsutil update. Do you think it is ok or maybe we should use `gsutil update -f` instead?

Reply:
It looks like we might even need to remove it completely, i just tested it locally and it shows:
```
It looks like you are trying to run "/usr/bin/../lib/google-cloud-sdk/bin/bootstrapping/gsutil.py update".
The "update" command is no longer needed with the Cloud SDK.
To update, run: gcloud components update
Really run this command? (y/N) 
```

Reply:
It seems with all these changes there is no point to do that. I will drop this line

Reply:
Is it ok like this?